docs/configuring-a-resource.md

```diff
@@ -557,13 +559,66 @@ p.AddResourceConfigurator("aws_autoscaling_group", func(r *config.Resource) {
 })
 ```
 
+### Initializers
+
+Initializers establish ownership of the supplied Managed resources in Crossplane.
+This typically involves the operations that are run before calling any
+ExternalClient methods.
+
+This configuration option will provide that setting custom initializers for per 
+resource and is set by using the [InitializerFns] field that is a list of [NewInitializerFn]:
+
+```go
+// NewInitializerFn returns the Initializer with a client.
+type NewInitializerFn func(client client.Client) managed.Initializer
+```
+
+Initializer is an interface in [crossplane-runtime]:
+
+```go
+type Initializer interface {
+	Initialize(ctx context.Context, mg resource.Managed) error
+}
+```
+
+So, an interface must be passed to the related configuration field for adding initializers for a resource.
+Let's check an example about how can this configuration be used.
+
+`tags` is very common schema field for aws resources. Also, setting [some external labels] is a very common
+use case in Crossplane. Therefore, an initializer that sets the crossplane related labels for resources
+can be added in config file of the related resource:
+
+```go
+// Tagger implements the Initialize function to set external tags
+type Tagger struct {
```

Comment:
Just a suggestion, you can possible refer to the common implementation in Terrajet as example and also something they can use right away if it works for them instead of implying that they need to write it on their own.